source/Mission.h

```diff
@@ -160,6 +161,10 @@ class Mission {
 	// Get a list of NPCs associated with this mission. Every time the player
 	// takes off from a planet, they should be added to the active ships.
 	const std::list<NPC> &NPCs() const;
+	// Get a list of timers associated with this mission.
+	std::list<Timer> &Timers();
```

Comment:
again I dont think we need a list here, a std set would be enough?
also we dont need this getter anymore right?

Reply:
> And am I correct in my understanding that the (primary?) difference between the two is that list can have duplicate members, while set cannot?
yes

> Again, I'm afraid I'm not well-acquainted with the distinctions between various STL collection types—I simply copied the usage for NPCs. Is it that there's something different about how Timers work than NPCs that means they should have a set while NPCs have a list, or would this also apply to the Mission's NPC collection?

dont know